kernel/nativeinstr.mli

```diff
@@ -23,7 +23,7 @@ and lambda =
   | Lrel          of Name.t * int 
   | Lvar          of Id.t
   | Lmeta         of metavariable * lambda (* type *)
-  | Levar         of existential * lambda (* type *)
+  | Levar         of Evar.t * lambda (* type *) * lambda array (* arguments *)
```

Comment:
I don't think you need the type anymore to perform reification, it is available through the evar map in `Nativenorm.nf_evar`.

Reply:
I think the idea is that we reduce the type.

Reply:
Well, you could do that separately, but it's probably more efficient the way you did it.